modules/concept-docs/pages/response-time-observability.adoc

```diff
@@ -4,7 +4,9 @@
 :page-topic-type: concept
 :page-aliases: ROOT:threshold-logging.adoc
 
+include::partial$attributes.adoc[]
+
 [abstract]
 {description}
 
-include::7.0@sdk:shared:partial$rto.adoc[tag=rto]
+include::{version-server}@sdk:shared:partial$rto.adoc[tag=placeholder]
```

Comment:
Think the tag should be `rto` here.
We changed this java for example: https://github.com/couchbase/docs-sdk-java/pull/241/files#diff-5e7cd414397b0cd64e652b9c2a31d46e7b02256472f33c1cd4523810e30e52d2R10

**SDK common:** https://github.com/couchbase/docs-sdk-common/blob/release/7.0/modules/shared/partials/rto.adoc

Reply:
huh... no idea where `placeholder` came from... possibly from another SDK docs copypasta... will fix here and make a note to check others...

good spot! 👍

Reply:
Probably because it was recently changed, `placeholder` was there before.